### ISSUE TITLE
Column Chooser - Suggestions

### ISSUE THREAD

**Issue by Quantas** [Quantas/shark-ng-table#65]
o   aria-expanded would help here as well, so screen reader users will know that they’ve expanded new content and can navigate to it. It would also be nice to make the Esc key close the menu and return focus to the menu button when the user is inside the menu – this will be more for sighted, keyboard users, unless you add all the aria semantics for a menu.

o   For these checkboxes, I think good labelling and a fieldset could remove the need for providing additional notification after each checkbox is toggled.

§  I would wrap these in a fieldset with a legend along the lines of “Columns to display”. You can give this the screen reader class so it doesn’t show up visually.

§  I would drop the title from the checkbox inputs, I think with a legend and the name of each column the intent should be pretty clear, and overall that will make it less “wordy”

§  Since checkboxes provide a notification when they are clicked (“checked”/”unchecked”), I think with fieldset legend it should be pretty clear what is going on here and no additional notification is needed. Though if you’ve already added one, I don’t think it will hurt.
